core-libs/setup/ssr/logger/loggers/default-server-logger.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LogContext, ServerLogger } from './server-logger';
+
+export class DefaultServerLogger extends ServerLogger {
```

Comment:
[MINOR]

What about name `PrerenderingServerLogger`? As far as I understand this logger is used only for prerendering?